### PR TITLE
Fix e2e tests, updated makefile

### LINKED ISSUE #141: Reenable end to end acceptance tests
Failing after #140

### PR DESCRIPTION
**Pull request by roll**
- fixes #141
- put all chore scripts to makefile

### CODE REVIEW

**Comment by roll** [comment]
@pwalsh 
@amercader 
So what's the point and my latest tries with npm/make - is preserve things manageable. So many moving parts for now with frontend/deployment etc so it's easy to finish with jungle for repo management (like forget to run e2e for travis or lost track of what during image creation process). So I'm +1 for any consistent system. Just think that for now we more probably figure out best naming etc for things inside one point of truth (makefile). And then we could move forward.

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.PHONY: help
+.PHONY: help frontend app-e2e
```
Comment (amercader) [inline Makefile]: Why is `frontend` PHONY?

Comment (roll) [inline Makefile]: Without it I'm getting `

Comment (amercader) [inline Makefile]: got it, because we have a `frontend` folder

Comment (roll) [inline Makefile]: Also I've just figure out why `frontend` acts like this - we have the dir with the same name.

Comment (pwalsh) [inline Makefile]: yep.

**Pull request merged**